authors/johnnie_oduro.md

```diff
@@ -0,0 +1,5 @@
+Author: Johnnie Oduro Jnr.
+Title: DevOps Engineer
+Description: Johnnie Oduro Jnr is a skilled DevOps Engineer with expertise in system monitoring, automation, and infrastructure as code. Currently exploring DevOps, SRE, and Data Analysis, Johnnie is committed to continuous learning and collaboration, backed by certifications in AWS and proficiency in Python, Linux, and Git.
+Company Name: BlueSPACE Africa Technologies
```

Comment:
do you have a logo? need bw for dark and light bg

Reply:
I'll check and include and add it.